### DISCUSSION TITLE
GitHub Copilot plugin 1.2.9.2684 is not support

### DISCUSSION THREAD

**Discussion by jasonhc** [community/community#110488] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm working with Intellij IDEA 2021.1.3 and copilot plugin v1.2.9.2684. Recently, the plugin not work and some error in log:
Your editor's Copilot extension, version "copilot-intellij/1.2.9.2684", is out of date. Please install a newer version to access this service. You may need to update your editor to install the newest extension version.

However, 1.2.9.2684 is latest version of Copilot plugin for IDEA 2021.1.3

**Comment by fgc1101** [comment]
> ### 选择主题领域
> 问题
> 
> ### 身体
> 我正在使用 Intellij IDEA 2021.1.3 和 copilot 插件 v1.2.9.2684。最近，该插件无法工作，日志中出现一些错误： 您编辑器的 Copilot 扩展，版本“copilot-intellij/1.2.9.2684”，已过期。请安装较新版本以访问此服务。您可能需要更新编辑器才能安装最新的扩展版本。
> 
> 但是，1.2.9.2684 是 IDEA 2021.1.3 的 Copilot 插件的最新版本

me too

**Comment by NikolasHofmann** [comment]
I have the same problem, tried reinstalling copilot, restarting pycharm etc.

**Reply by NikolasHofmann** [reply]
Quick update:
I uninstall copilot, uninstall pycharm and then reinstalled the latest pycharm community version (I was using a 2021 version before).

This has completely solved my issue.

**Comment by Zdunek** [comment]
PHPStorm, same problem...

**Comment by desperateBob** [comment]
Faced the same issue today and it's still not working.

**Reply by Kinzel** [reply]
Same here

**Comment by iamroberthanson** [comment, marked as answer]
Same here.  It started happening on the afternoon of February 27th 2024.

Update: I was running Idea Community Edition from 2021.  I upgraded the the latest Community Edition version (2023.3.4) and Copilot is working again.

**Reply by jasonhc** [reply]
Thank for your quick answer. However, I'd like use IDEA 2021 because version 2023 run slow on my computer.

**Comment by someramsey** [comment]
Same. 
I am using co-pilot for both vscode and Intellij-idea. It seems to be working fine on VScode but not on Intellij-idea I get the plugin out of date error

> Error: Your editor's Copilot extension, version "copilot-intellij/l .2.9.2684", is out of date. Please install a newer version to access this service. You may need to update your editor to install the newest extension version


Update:

Upgrading Intellij to version 2023 has fixed the issue for me.

**Comment by desperateBob** [comment]
It works again on my Intellij 2021.1.3. I hope it won't be an issue anymore.

**Comment by Jenverse** [comment]
We've re-enabled this(1.2.9) version for now so that IntelliJ 2021 users can continue using Copilot but in general we do recommend staying up to date as we don't commit to supporting older versions

**Comment by Yosefjos** [comment]
![IMG_20240404_220937_359](https://github.com/community/community/assets/165850277/0333564f-a43d-4b4a-a846-634e4f3500ce)
